dts/common/yaml/uart.yaml

```diff
@@ -27,4 +27,14 @@ properties:
       category: required
       description: Human readable string describing the device (used by Zephyr for API name)
       generation: define
+  - interrupts-name:
+      type: array
```

Comment:
Do you mean "interrupt-names"?
The interrupt-names property consists of a list of strings so the "stringlist" should be used here.